dashboard/charts.py

```diff
@@ -90,7 +95,7 @@ def get_albums_sales_line_graph(chosen_album) -> alt.Chart:
 @st.cache_data
 def get_tag_sales_line_graph(chosen_tag) -> alt.Chart:
     """Returns a line graph of sales over time for a chosen tag"""
-    return alt.Chart(chosen_tag).mark_line(point=True).encode(
+    return alt.Chart(chosen_tag).mark_line(point=True, color="rgb(60, 154, 170)").encode(
```

Comment:
Could have this as a global constant so that it's easier to find

Reply:
Yep good suggestion